airbyte-integrations/connectors/source-mongodb-v2/src/main/java/io/airbyte/integrations/source/mongodb/cdc/MongoDbCdcInitializer.java

```diff
@@ -80,8 +79,14 @@ public List<AutoCloseableIterator<AirbyteMessage>> createCdcIterators(
                                                                         final Instant emittedAt,
                                                                         final MongoDbSourceConfig config) {
 
-    final Duration firstRecordWaitTime = RecordWaitTimeUtil.getFirstRecordWaitTime(config.rawConfig());
-    final Duration subsequentRecordWaitTime = RecordWaitTimeUtil.getSubsequentRecordWaitTime(config.rawConfig());
+//    LOGGER.info("*** config {}", config.rawConfig());
+    final Duration firstRecordWaitTime = Duration.ofSeconds(config.getInitialWaitingTimeSeconds());
+    // #35059: debezium heartbeats are not sent on the expected interval. this is a worksaround to allow
+    // making
+    // subsequent wait time configurable.
+    // final Duration subsequentRecordWaitTime = firstRecordWaitTime.dividedBy(2);
```

Comment:
Is the goal to make the subsequent record wait time half of the first record wait time or the same?